bedrock/firefox/templates/firefox/australis/firstrun.html

```diff
@@ -59,7 +59,12 @@
     </ul>
     <ul>
       <li><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-name="Legal">{{ _('Legal') }}</a></li>
-      <li><a href="{{ url('privacy') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
+      {% if LANG == 'en-US' %} 
```

Comment:
This is a legacy page that is not indexed by search engines, and I doubt we would ask localizers to update it at this point. This does no harm, but it will probably remain like this until the page is decomissioned (you choice here).

Reply:
Understood. I'm just trying to be thorough.